packages/components/molecules/f-searchbox/README.md

```diff
@@ -227,3 +227,47 @@ const copyOverrides = {
     // ...
 }
 ```
+
+### Non-Vue Applications
+
+This module can be ran as a micro front-end for applications that don't make use of the Vue framework.
+
+The following rudimentary example can be used as a guide for implementing this component in an existing static application.
+
+NOTE: You need to provide Vue with an instance of Vuex for f-searchbox to work correctly. This should be
+added after Vue has been included. Including the `unpkg` Vuex script, like the below example, will automatically install Vuex.
+
+The version of Vuex required must be no earlier than `vuex@3.2.0` as f-searchbox under the hood registers
+the store module using `this.$store.hasModule('searchbox')`, `hasModule` was added in `v3.2.0`, earlier Vuex versions will break store registration.
+
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <title>f-searchbox example standalone</title>
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/f-searchbox/dist/f-searchbox.css">
+</head>
+<body>
+    <div data-app>
+        <vue-Searchbox locale="en-GB" />
```

Comment:
Ill lowercase the `S` in my current pr. missed it :\

Reply:
Ah yeah, I missed that too

Reply:
Fixed https://github.com/justeat/fozzie-components/pull/651/files#diff-702ba200f743792e5005b0e1e0c5dd447c3569ca61dc0d29a6c0fde0408f44b7R253